backend/learning_app/models.py

```diff
@@ -17,3 +17,11 @@ class StudentFollowInformation(models.Model):
         null=True,
     )
     is_following = models.BooleanField(default=True)
+
+
+class Category(models.Model):
+    title = models.CharField(max_length=50)
+    description = models.CharField(max_length=250)
```

Comment:
TextField is more preferable for description fields.
```suggestion
    description = models.TextField(max_length=250)
```